src/main/java/com/sololegends/runelite/VarlamoreHouseThievingPlugin.java

```diff
@@ -199,6 +199,13 @@ public void onGameTick(GameTick event) {
 		} else {
 			done_stealing_notified = false;
 		}
+
+		// Time since last distraction notification check
+		long timeSinceLastDistraction = NextUpOverlayPanel.sinceDistraction();
+
+		if (config.notifyOnTimeSinceDistraction() == timeSinceLastDistraction && config.notifyOnTimeSinceDistraction() > 0) {
+			notify("It has been " + timeSinceLastDistraction + " seconds since the last distraction");
```

Comment:
Should account for crazy people putting in 1 second and use the proper plurality.